packages/tools/kadena-cli/src/account/commands/accountFund.ts

```diff
@@ -96,7 +96,10 @@ export const createAccountFundCommand = createCommand(
 
         const { deployFaucet } = await option.deployFaucet();
 
-        if (deployFaucet === false) {
+        if (!deployFaucet) {
+          log.info(
+            'Skipping faucet deployment... If you would like to deploy the faucet, please re-run the command with the "--deploy-faucet" flag.',
```

Comment:
The wording seems a bit odd. it says `skipping faucet` as if it will continue to do something else.
but if faucet needs to be deployed and you choose not to, nothing else can happen right?

Reply:
yes, nothing else can happen, it's just exit.